monkey/monkey_island/cc/resources/auth/register.py

```diff
@@ -23,20 +24,24 @@ def __init__(self, authentication_service: AuthenticationService):
 
     def post(self):
         """
-        Registers a new user
+        Registers a new user using flask security register
 
-        Gets a username and password from the request sent from the client,
-        and registers a new user
-
-        :raises InvalidRegistrationCredentialsError: If username or password is empty
-        :raises AlreadyRegisteredError: If a user has already been registered
         """
-
-        username, password = get_username_password_from_request(request)
-
         try:
-            self._authentication_service.register_new_user(username, password)
-            return make_response({"error": ""}, HTTPStatus.OK)
-        # API Spec: HTTP status code for AlreadyRegisteredError should be 409 (CONFLICT)
-        except (InvalidRegistrationCredentialsError, AlreadyRegisteredError) as e:
-            return make_response({"error": str(e)}, HTTPStatus.BAD_REQUEST)
+            username, password = get_username_password_from_request(request)
+            if not username or not password:
+                return make_response(
+                    {"error": "Provided empty credentials"}, HTTPStatus.BAD_REQUEST
+                )
+
+            # This method take the request data and pass it to the RegisterForm
+            # where a registration request is preform.
+            # Return value is a flask.Response object
```

Comment:
We don't need to describe what `register` does, it's described under the comments of `register`